apps/wallet/src/background/transactions.ts

```diff
@@ -156,9 +153,12 @@ class Transactions {
         origin: string,
         favIcon?: string,
     ) {
+        const requestingAccount =
```

Comment:
this var holds just the address so maybe we can rename it to `requestingAddress`